CONTRIBUTING.md

```diff
@@ -4,10 +4,11 @@
 
 ### Configuring python environment
 
-The project requires Python version 3.7+. To set it as default in the environment run the following commands:
+The project development requires Python version 3.9+. To set it as default in the environment run the following commands:
 
 ```sh
-python -m venv env
+sudo apt-get install python3.9-venv
+python3.9 -m venv env
```

Comment:
I would not make them apt specific. I personally use e.g. pyenv to install Python 3.9 and then venv to create a virtual environment.

Reply:
Put this into a comment. I'm always puzzled at this step when I get to playwright python.

Reply:
fair enough!